src/App.jsx

```diff
@@ -14,57 +15,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGoods, setSelectedGoods] = useState(['Jam']);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const addGood = good => {
+    if (!selectedGoods.includes(good)) {
+      setSelectedGoods([...selectedGoods, good]);
+    }
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const removeGood = good => {
+    setSelectedGoods(selectedGoods.filter(item => item !== good));
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      {selectedGoods.length === 0 ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        selectedGoods.map(selectedGood => (
+          <h1
+            key={selectedGood}
+            className="title is-flex is-align-items-center"
+          >
+            {selectedGood} is selected
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
+              className="delete ml-3"
+              onClick={() => removeGood(selectedGood)}
+            />
```

Comment:
The `<button>` element should not be self-closing. In HTML, `<button>` elements should have an opening and a closing tag. Change `<button ... />` to `<button ... ></button>` to ensure proper rendering.